fenixedu-ist-registration-process/src/main/java/pt/ist/registration/process/ui/SignedDeclarationsController.java

```diff
@@ -72,8 +72,7 @@ public String certifierCallback(@PathVariable Registration registration, @Reques
             try {
                 logger.debug("Registration Declaration {} of student {} was certified.", uniqueIdentifier, registration.getNumber());
                 logger.debug("Registration Declaration {} of student {} sent out to be stored.", uniqueIdentifier, registration.getNumber());
-                signCertAndStoreService.sendDocumentToBeStored(registration.getPerson().getUsername(),
-                        registration.getPerson().getEmailForSendingEmails(), declarationFile.getFilename(), file, uniqueIdentifier);
+                signCertAndStoreService.sendDocumentToBeStored(registration.getPerson().getUsername(), registration.getPerson().getEmailForSendingEmails(), declarationFile, file);
```

Comment:
This line does not compile:
SignedDeclarationsController.java:[75,39] error: method sendDocumentToBeStored in class SignCertAndStoreService cannot be applied to given types;